server/routes/serviceProviderReferrals/addActionPlanActivitiesPresenter.test.ts

```diff
@@ -46,16 +46,45 @@ describe(AddActionPlanActivitiesPresenter, () => {
 
   describe('text', () => {
     describe('title', () => {
-      it('includes the name of the service category', () => {
-        const socialInclusionServiceCategory = serviceCategoryFactory.build({ name: 'social inclusion' })
-        const actionPlan = actionPlanFactory.justCreated(sentReferral.id).build()
-        const presenter = new AddActionPlanActivitiesPresenter(
-          sentReferral,
-          [socialInclusionServiceCategory],
-          actionPlan
-        )
+      describe('when no activities have been added', () => {
+        it('specifies the number of the activity to be added', () => {
+          const socialInclusionServiceCategory = serviceCategoryFactory.build({ name: 'social inclusion' })
+          const actionPlan = actionPlanFactory.justCreated(sentReferral.id).build()
+          const presenter = new AddActionPlanActivitiesPresenter(
+            sentReferral,
+            [socialInclusionServiceCategory],
+            actionPlan
+          )
+
+          expect(presenter.text.title).toEqual('Add activity 1 to action plan')
+        })
+      })
 
-        expect(presenter.text.title).toEqual('Social inclusion - create action plan')
+      describe('when 1 activity has been added', () => {
+        it('specifies the number of the activity to be added', () => {
+          const socialInclusionServiceCategory = serviceCategoryFactory.build({ name: 'social inclusion' })
+          const actionPlan = actionPlanFactory.justCreated(sentReferral.id).build({
+            activities: [
+              {
```

Comment:
This `Activity` object contains a lot of data that's not relevant for this test case, which is only concerned with the number of activities in the action plan.

It might be a good idea to create an `actionPlanActivityFactory`, so that we can create valid instances of `Activity` in a single line. We'd also want to update `actionPlanFactory` to call this new factory when creating activity instances.

Reply:
Good spot, I've added an `activityFactory` for this and used it in those tests.